mistletoe/ast_renderer.py

```diff
@@ -33,9 +33,11 @@ def get_ast(token):
     #   [1]: https://docs.python.org/3/whatsnew/3.6.html
     #   [2]: https://github.com/syntax-tree/mdast
     node['type'] = token.__class__.__name__
-    node.update(token.__dict__)
-    if 'header' in node:
-        node['header'] = get_ast(node['header'])
-    if 'children' in node:
-        node['children'] = [get_ast(child) for child in node['children']]
+    for attr_name in ['content', 'header', 'footnotes']:
```

Comment:
Good idea for DRY, unfortunately it looks like we still need to call `get_ast(node.header)` - storing just `getattr(token, 'header')` throws an error.

BTW Rendering of Tables to AST is not covered by the current unit tests - do you think you could extend them while at it, please? :)

Reply:
Done. Wish the unit test had been there in the first case :)

Reply:
Near perfect now, thanks. :)